test/public/runs/runsPerPeriod.overview.test.js

```diff
@@ -171,18 +170,16 @@ module.exports = () => {
             // eslint-disable-next-line no-undef
             model.runs.perLhcPeriodOverviewModel.pagination.itemsPerPage = 10;
         });
-        await waitForTimeout(100);
+        await waitForTableLength(page, 3);
     });
 
     it('can navigate to a run detail page', async () => {
-        await reloadPage(page);
-        await waitForTimeout(100);
+        await waitForNavigation(page, () => reloadPage(page));
```

Comment:
Same, the best is to simply avoid using reloadPage if you can

Reply:
Done